s3same/__init__.py

```diff
@@ -0,0 +1,37 @@
+import boto3
+import travispy
+import yaml
+
+from .travis import travis_encrypt as _travis_encrypt
```

Comment:
Yeah, that's what I was trying to ask.  I didn't know if using `__all__` was enough of a convention or if `_name` or `__name` were more of a thing in this case.